src/lib/y2firewall/proposal_settings.rb

```diff
@@ -59,6 +59,7 @@ def initialize
       # chose a different one in the proposal
       @default_zone = "public"
       @selinux_config = Y2Security::SelinuxConfig.new
+      @selinux_config.save # lets write the proposal to sync the initial state
```

Comment:
But current implementation of [SelinuxConfig#save](https://github.com/yast/yast-security/blob/2ccd5c23b7bc1f6fd77d6b9e8ff55d199a1b6258/src/lib/y2security/selinux_config.rb#L128-L148) will not write the initial state unless [SelinuxConfig#configurable?](https://github.com/yast/yast-security/blob/2ccd5c23b7bc1f6fd77d6b9e8ff55d199a1b6258/src/lib/y2security/selinux_config.rb#L150-L159)

Is that ok?

Reply:
yes, it is exactly what I need. As if it is not configurable, we do not touch it, so it is basically nothing and we do not display anything, neither write.